### PR TITLE
Pdf

### PR DESCRIPTION
**Pull request by mayrapena1324**
I added the pdf capability!  I also tried to add a new editor and epically failed.

### CODE REVIEW

File: app/error_handlers.py [hunk 1]
```diff
@@ -7,3 +7,10 @@
 def forbidden(e):
     app.logger.error(f"Forbidden Access{e}, route: {request.url}")  # pass in the error itself and the error
     return render_template("public/error_handlers/403.html")
+
+
+# Not Found Error
+@app.errorhandler(404)
+def forbidden(e):
```
Comment (barrywhart) [inline app/error_handlers.py]: Should this function be renamed, e.g. as `not_found()`? At best, it's confusing to have two functions named `forbidden` in the same file.

File: app/templates/user/all_rehearsals.html [hunk 2]
```diff
@@ -32,11 +32,11 @@ <h2>Today is {{today}}</h2>
         {% for group in distinct_groups %}
         <div class="col mb-4">
             <div class="card shadow p-3 mb-5 bg-white rounded">
-
```
Comment (barrywhart) [inline app/templates/user/all_rehearsals.html:35]: Nit: The changes to this file are all whitespace. Are they needed for the PDF functionality? (Not a big deal, but sometimes in open-source or professional work, it's useful to minimize changes and keep them focused on what's necessary. Sometimes reviewers will give you a hard time one way or another when they see unrelated changes.)

File: app/error_handlers.py (hunk 1 above)
Comment (mayrapena1324) [reply app/error_handlers.py]: I  just got caught copying and pasting. 😅

File: app/templates/user/all_rehearsals.html (hunk 2 above)
Comment (mayrapena1324) [reply app/templates/user/all_rehearsals.html:35]: This is good to know. How can I remove them from the PR? Do I just remove the line?

File: app/error_handlers.py (hunk 1 above)
Comment (barrywhart) [reply app/error_handlers.py]: I know! 😉

File: app/templates/user/all_rehearsals.html (hunk 2 above)
Comment (barrywhart) [reply app/templates/user/all_rehearsals.html:35]: You can manually undo the changes one by one, or another technique is to `git checkout master`, copy the file somewhere, then `git checkout pdf`, and copy the file over, then commit it. Semi-automated "undo"!
